services/logging-operator/4.2.3/extra-images.txt

```diff
@@ -0,0 +1,4 @@
+{{ .Values.fluentbit.image.repository }}:{{ .Values.fluentbit.image.tag }}
+{{ .Values.fluentd.image.repository }}:{{ .Values.fluentd.image.tag }}
+ghcr.io/kube-logging/config-reloader:v0.0.5
```

Comment:
for images like this, it looks like we'll need to pay more attention to bumping them now. 🤔

some of these images are [hardcoded in the operator](https://github.com/kube-logging/logging-operator/blob/b343ee8b58388206ffd56cc55a2cf2bc57115ec7/pkg/sdk/logging/api/v1beta1/logging_types.go#L144) so we have to now check the operator codebase to ensure this list is up to date. I wonder if there's a way we can make sure this is caught prior to merging PRs right now.

previously, i think these would be caught when running the logging operator in a cluster so we didn't really need to consider it.

Reply:
yeah deploying this one off app and getting the images doesn't seem worthwhile. 

wdyt about just adding a comment so someone bumping the logging operator could check if these images changed?

side note: 

it actually looks like these could be set in the helm chart values, we could do that in the future then just have to worry about bumping them manually (instead of having missing images, they might just be out of date)

https://kube-logging.dev/docs/configuration/crds/v1beta1/fluentd_types/#fluentdspec-configreloaderimage
https://kube-logging.dev/docs/configuration/crds/v1beta1/fluentd_types/#fluentdspec-buffervolumeimage

Reply:
Although you can set them via the chart, that only sets the values as flags to the operator and so wouldn't be picked up here.